Source/items.cpp

```diff
@@ -2504,8 +2504,8 @@ void CheckIdentify(int pnum, int cii)
 {
 	ItemStruct *pi;
 
-	if (cii >= 7)
-		pi = &plr[pnum].InvList[cii - 7];
+	if (cii >= NUM_INVLOC)

```

Comment:
This not having been applied really annoying me recently when implementing the controller code :)